sor/src/test/java/com/bazaarvoice/emodb/sor/core/CompactionControlTest.java

```diff
@@ -47,15 +49,15 @@ public void allDeltasBeforeCompactionControlTimestampShouldNotBeDeleted() {
         Delta delta2 = Deltas.literal(ImmutableMap.of("key2", "value2"));
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "value2" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.qa.bazaarvoice.com/coding_rules#rule_key=squid%3AS1192)